tests/test_qpu_devices.py

```diff
@@ -290,6 +291,99 @@ def test_check_connection_provider_wrong_backend_provided_credentials(self):
         self.assertEqual(device_obj.check_connection(), False)
         self.assertEqual(device_obj.provider_connected, True)
         self.assertEqual(device_obj.qpu_connected, False)
+        
+
+class TestingDeviceAzure(unittest.TestCase):
+    
+    """These tests check the Object used to access Azure and their 
+    available QPUs can be established.
+
+    For any tests using provided credentials, the tests will only pass if those
+    details provided are correct/valid with Azure.
+    """
+    
+    @pytest.mark.api
+    def setUp(self):
+
+        try:
```

Comment:
Why are we testing things with the credentials file?

Reply:
Removed